### ISSUE TITLE
Get TPM manufacturer / tests

### ISSUE THREAD

**Issue by lkatalin** [keylime/rust-keylime#319]
Not sure if applicable: https://github.com/keylime/keylime/pull/870

**Comment by THS-on** [comment]
I don't think so because we are using `get_tpm_vendor()` ( https://docs.rs/tss-esapi/latest/tss_esapi/utils/fn.get_tpm_vendor.html) from the tss library and not parsing the output of `tpm2_getcap` manually.
